features/step_definitions/consultation_steps.rb

```diff
@@ -6,8 +6,10 @@
   create(:unopened_consultation)
 end
 
-When(/^I draft a new consultation "([^"]*)"$/) do |title|
-  begin_drafting_document type: "consultation", title: title, summary: "consultation-summary", alternative_format_provider: create(:alternative_format_provider), all_nation_applicablity: false
+When(/^I draft a new "(.*?)" language consultation "(.*?)"$/) do |locale, title|
+  document_options = { type: "consultation", title: title, summary: "consultation-summary", alternative_format_provider: create(:alternative_format_provider), all_nation_applicablity: false }
+  document_options.merge!(locale: locale) unless locale == "English"
```

Comment:
Looks like you don't need the `unless` anymore, as you've specified the locale in all the feature tests.